### DISCUSSION TITLE
2022 SIParCS Brainstorming

### DISCUSSION THREAD

**Discussion by kmpaul** [ProjectPythia/projectpythia.github.io#229] [Ideas]
The 2022 SIParCS internships will be starting soon!  We have a proposed project description below:

> Project Pythia (https://projectpythia.org/) provides a public, web-accessible training resource to help educate current and aspiring earth scientists to more effectively use both the Scientific Python Ecosystem and Cloud Computing to make sense of large volumes of numerical scientific data. We are looking for help developing, curating, and testing our content. This involves finding external learning resources to add to the Pythia Resource Gallery (https://projectpythia.org/gallery.html), working through our resources on the Pythia Foundations Book (https://foundations.projectpythia.org/) while documenting your learning experience, and writing more documentation on various Python tools, packages, and techniques to add to the Pythia Foundations Book. If the student is interested, work can be expanded to include outreach efforts to help grow our audience, help with tutorial workshops or other events, and potentially some web development.

In terms of *specific* things for the intern to work on, what ideas do you have?

**Comment by jukent** [comment]
Welcome James Morley!

Here is a compilation of good first projects:

Read through the [Pythia ](https://projectpythia.org/)foundations book
Learn foundational skills
Provide feedback on the content (where does it not flow, where are there gaps in explanations, or questions you have)

Follow tutorials in tutorial series

Look for Pythia foundations [issues](https://github.com/ProjectPythia/pythia-foundations/issues)
[Markdown tutorial](https://github.com/ProjectPythia/pythia-foundations/issues/232)
[Add links to Foundations content](https://github.com/ProjectPythia/projectpythia.github.io/issues/104)
[Quickstart add links for further study](https://github.com/ProjectPythia/pythia-foundations/issues/64)
[Matplotlib consistent colorbars](https://github.com/ProjectPythia/pythia-foundations/issues/276)
[Advanced Matplotlib content](https://github.com/ProjectPythia/pythia-foundations/issues/239)
[Overview of Software Stack](https://github.com/ProjectPythia/pythia-foundations/issues/220)

While working on addressing any of the above issues, evaluate the Pythia contributor’s guides [here](https://foundations.projectpythia.org/appendix/how-to-contribute.html) and here. Do these guides, along with the “Getting Started” material in Foundations, give enough information to be able to open PRs and make improvements? Is t[here](https://projectpythia.org/contributing.html) material in the guides that is inconsistent with Foundation [Getting started with GitHub](https://foundations.projectpythia.org/foundations/getting-started-github.html)?  Can they be better cross-referenced?
